### ISSUE TITLE
Mojaloop: Create self-service user associated with Fineract customers in PH-EE Lab Environment 

### ISSUE THREAD

**Issue by edcable** [openMF/mobile-wallet#978]
In order to support the first two use case for Mojaloop integration via our customer-facing channels through the Payment Hub EE we need to actually authenticate and conduct transactions via self-service APIs. Self-service users need to be associated with the customers/merchants in the medium Fineract instance that have MSISDNs:

Customer 1
http://leopard.mifos.io:9002/?baseApiUrl=https://leopard.mifos.io:8443&tenantIdentifier=tn05

Name: InteropCustomer
Account Number: 830bbcd0d6a044e38c6631
Client ID: 1

Merchant 1

http://gorilla.mifos.io:9002/?baseApiUrl=https://gorilla.mifos.io:8443&tenantIdentifier=tn06

MSISDN: 27710306999

Name: InteropMerchant
Account Numberb780411350952999709451

**Comment by edcable** [comment]
@ezolnbl thank you for upgrading instance of Fineract. I was able to create a self-service user through the UI.

@shiv07tiwari @devansh-299 are you able to test and see if you can log in with the self service user of walletuser/password that is associated with 

Customer 1
http://leopard.mifos.io:9002/?baseApiUrl=https://leopard.mifos.io:8443&tenantIdentifier=tn05

Name: InteropCustomer
Account Number: 830bbcd0d6a044e38c6631
Client ID: 1

**Comment by edcable** [comment]
I added user merchantuser/password for the merchant customer:

Merchant 1

http://gorilla.mifos.io:9002/?baseApiUrl=https://gorilla.mifos.io:8443&tenantIdentifier=tn06

MSISDN: 27710306999

Name: InteropMerchant
Account Numberb780411350952999709451

**Comment by devansh-299** [comment]
> @ezolnbl thank you for upgrading instance of Fineract. I was able to create a self-service user through the UI.
> 
> @shiv07tiwari @devansh-299 are you able to test and see if you can log in with the self service user of walletuser/password that is associated with
> 
> Customer 1
> http://leopard.mifos.io:9002/?baseApiUrl=https://leopard.mifos.io:8443&tenantIdentifier=tn05
> 
> Name: InteropCustomer
> Account Number: 830bbcd0d6a044e38c6631
> Client ID: 1

Yes @edcable I the credentials are working fine. Tested via Postman

**Comment by therajanmaurya** [comment]
Moved on jira https://mifosforge.jira.com/browse/MW-196
